src/forms/PhoneField/PhoneField.test.jsx

```diff
@@ -0,0 +1,105 @@
+import {render, fireEvent} from '@testing-library/preact';
+
+import PhoneField from './PhoneField';
+
+describe('UI/forms/PhoneField', () => {
+    const defaultProps = {
+        placeholder: 'placeholder',
+        countryCode: 'MX'
+    }
+
+    it('Should not show error label if given value is empty', () => {
+        const {getByTestId} = render(<PhoneField  {...defaultProps} />);
+        const noErrorLabel = !getByTestId('phone-field-label').className.includes('error');
+        expect(noErrorLabel).toBeTruthy();
+    })
+
+    it('Should not show error label if given value is valid', () => {
+        const {getByTestId} = render(<PhoneField  {...defaultProps} value='5512341234'/>);
+        const noErrorLabel = !getByTestId('phone-field-label').className.includes('error');
+        expect(noErrorLabel).toBeTruthy();
+    });
+
+    it('Should show error label if given value is NOT valid', () => {
+        const {getByTestId} = render(<PhoneField  {...defaultProps} errorLabel={'Error!'} value='551234123234324'/>);
+        const errorLabel = getByTestId('phone-field-label').className.includes('error');
+        expect(errorLabel).toBeTruthy();
```

Comment:
Could you check if it is `Error!` what's displayed, as well?

Reply:
Nice one! sure :) thanks!